src/olympia/reviewers/tests/test_views.py

```diff
@@ -5632,6 +5632,14 @@ def test_review_moderator_addon_rating_present(self):
         )
         assert doc('.addon-rating a').attr['href'] == rating_url
 
+        self.reviewer.update(email='reviewer@nonmozilla.com')
+
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        doc = pq(response.content)
+        assert doc('.addon-rating strong')
```

Comment:
I just tried this locally and unfortunately this would pass anyway without your changes - there's a `<strong>` inside the link too.  We need something that checks for the `<a..>` really - e.g. `assert not doc('.addon-rating a')`